src/deserializer.rs

```diff
@@ -297,10 +301,14 @@ impl<'r> DeRecord<'r> for DeByteRecord<'r> {
             visitor.visit_bool(true)
         } else if x == b"false" {
             visitor.visit_bool(false)
-        } else if let Some(n) = try_positive_integer_bytes(x) {
+        } else if let Some(n) = try_positive_integer_bytes_64(x) {
             visitor.visit_u64(n)
-        } else if let Some(n) = try_negative_integer_bytes(x) {
+        } else if let Some(n) = try_negative_integer_bytes_64(x) {
             visitor.visit_i64(n)
+        }else if let Some(n) = try_positive_integer_bytes(x) {
+            visitor.visit_u128(n)
+        } else if let Some(n) = try_negative_integer_bytes(x) {
+            visitor.visit_i128(n)
```

Comment:
If you _replace_ the 64-bit parsing with 128-bit parsing completely, do the tests still pass?

If not, then this is fine I think!

Reply:
It does break two existing tests. I'm trying to work out why to see if I can remove the 64-bit parsing fully.

Reply:
Figured. I don't think you can get rid of. Serde will "cascade" the 8/16/32 bit integers up to u64 automatically, which is why we don't need the smaller sizes. But 128-bit integers are a conditional feature since they are somewhat new, so there's no cascading from 64 bit integers up to 128-bit integers. I figured this was the case, but wasn't 100%.

I think this pretty much looks good to go. I'm going to hold off on merging it until I'm ready to release `csv 1.1.0` though. (Because this increases the MSRV.)